.github/workflows/pull.yml

```diff
@@ -239,6 +239,36 @@ jobs:
         # see if we can import the module successfully
         python -c "from executorch.extension.pybindings import portable_lib; print('success!')"
 
+  test-binary-size-linux:
+    name: test-binary-size-linux
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    strategy:
+      fail-fast: false
+    with:
+      runner: linux.2xlarge
+      docker-image: executorch-ubuntu-22.04-clang12
+      submodules: 'true'
+      ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
+      timeout: 90
+      script: |
+        # The generic Linux job chooses to use base env, not the one setup by the image
+        CONDA_ENV=$(conda env list --json | jq -r ".envs | .[-1]")
+        conda activate "${CONDA_ENV}"
+
+        # build module for executorch.extension.pybindings.portable_lib
+        bash test/build_size_test.sh
+        strip cmake-out/test/size_test
+        output=$(ls -la cmake-out/test/size_test)
+        arr=($output)
+        size=${arr[4]}
+        threshold="48120"
```

Comment:
Is this the current size?

Reply:
For my local on devserver, yes. On CI it is 51768

Reply:
ah, seems the difference is gcc vs clang